src/main/java/net/sf/jabref/LoadedDatabase.java

```diff
@@ -0,0 +1,73 @@
+package net.sf.jabref;
+
+import net.sf.jabref.model.database.BibDatabase;
+import net.sf.jabref.model.database.BibDatabaseType;
+import net.sf.jabref.model.database.BibDatabaseTypeDetection;
+
+import java.io.File;
+import java.util.Objects;
+import java.util.Vector;
+
+public class LoadedDatabase {
+
+    private static final String DATABASE_TYPE = "DATABASE_TYPE";
+
+    private final BibDatabase database;
+    private final MetaData metaData;
+
+    public LoadedDatabase() {
```

Comment:
So LoadedDatabase is just a temporary construct and will disappear in the long term?